.github/actions/push-react-artefact/action.yaml

```diff
@@ -0,0 +1,31 @@
+name: "Upload react app artefact"
```

Comment:
I will channel my inner JP ... the name of actions should end with action - eg "Upload react app artefact action"

Reply:
Makes sense have changed it to - "Upload react app artefact action"